src/js/components/search/table/ResultsTable.jsx

```diff
@@ -54,7 +54,11 @@ export default class ResultsTable extends React.Component {
         const columnId = this.props.columns.visibleOrder[columnIndex];
         const column = this.props.columns.data[columnId];
         const isLast = (columnIndex + 1) === this.props.columns.visibleOrder.length;
-        const isActive = this.props.sort.field === column.columnName;
+        let isActive = this.props.sort.field === column.columnName;
+
+        if (!isActive && column.columnName === 'Action Date' && this.props.sort.field === 'Sub-Award Date') {
```

Comment:
the ui functions correctly but I'm wondering why we still have a prop named "Sub-Award Date".  If we could change this prop to "Action Date" it would make this code much simpler.

Reply:
do you know where the sub-award date name is being applied to the api response?